spec/models/webauthn_credential_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe WebauthnCredential do
```

Comment:
General point: I don't think we should test that factories are valid - that should be covered (indirectly) by other tests which use the factories.

Reply:
@AlanGabbianelli and I had a chat about this one of our first days pairing.
We'd spoken about how in other repos we'd not done this within a model test, but rather had a separate test that just iterates over all factories and variations in traits and tests for validity. That should be less code and a better genrealised test than what we did here.

I do think there's value in it though, I could imagine scenarios where we've run a migration change and not updated the factories. Ideally that would get picked up in the tests like you say, but possible not?

And these are essentially low cost, quick unit tests, so I don't see much harm?